isotovideo

```diff
@@ -120,6 +120,11 @@ $bmwqemu::vars{BACKEND} ||= "qemu";
 # This allows further structuring the test distribution collections with
 # multiple distributions or flavors in one repository.
 $bmwqemu::vars{PRODUCTDIR} ||= $bmwqemu::vars{CASEDIR};
+
+# as we are about to load the test modules store the git hash that has been
+# used. If it is not a git repo fail silently, i.e. store an empty variable
+chomp($bmwqemu::test_git_hash = qx{git rev-parse HEAD});
```

Comment:
hm, it "magically" found the wrong repo of os-autoinst not the tests, obviously.

updated:
 * fix to go to actual test directory
 * writing to `vars.json` instead which is stored as result file within openQA jobs and as `vars.json` is changed by the test anyway